index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Hamma's Portflolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <nav class="nav-bar">
+            <p>Flex Box</p>
+            <img src="img/Enabled.png" alt="Logo">
+        </nav>
```

Comment:
- [ ] Kindly add `header` semantic tag instead of `nav` and use `nav` for wrap burger icon image to improve SEO for your site. You can read more form [here](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) about semantic tags
- [ ] Please, use `<a>` for your logo `Flex box` because this link will point to the header of your site